hk2-testing/hk2-mockito/src/test/java/org/jvnet/testing/hk2mockito/IndirectCollaboratorMockInjectionTest.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright (c) 2014, 2018 Oracle and/or its affiliates. All rights reserved.
```

Comment:
If you don't work for Oracle then this should be yourself instead, unless this file is a rename from elsewhere that git hasn't picked up on.